akka-docs/rst/java/distributed-data.rst

```diff
@@ -444,6 +444,49 @@ look like for the ``TwoPhaseSet``:
 
 .. includecode:: code/docs/ddata/japi/protobuf/TwoPhaseSetSerializer2.java#serializer
   
+Durable Storage
+---------------
+
+By default the data is only kept in memory. It is redundant since it is replicated to other nodes 
+in the cluster, but if you stop all nodes the data is lost, unless you have saved it 
+elsewhere. 
+
+Entries can be configured to be durable, i.e. stored on local disk on each node. The stored data will be loaded
```

Comment:
Maybe also something like "this means data will survive as long as at least one node from the old cluster takes part in a new cluster" ?